extensions/security/runtime-spi/src/main/java/io/quarkus/security/spi/runtime/SecurityHandlerConstants.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.security.spi.runtime;
+
+public class SecurityHandlerConstants {
+
+    /**
+     * Invocation context data key used by the SecurityHandler to save a security checks state
+     */
+    public static final String SECURITY_HANDLER = "SECURITY_HANDLER";
```

Comment:
The PR looks good but I wonder if the `SECURITY_HANDLER` constant is "unique enough". Usually it's good to use a package prefix so that it's immediately clear where the binding comes from, i.e. `io.quarkus.security.securityHandler`...

Reply:
Your example is way better than `SECURITY_HANDLER`, I used it, thank you.